modules/nodes-scheduler-node-selectors-cluster.adoc

```diff
@@ -39,9 +39,14 @@ spec:
 
 If the project where you are creating the pod has a project node selector, that selector takes preference over a cluster node selector.
 
+[NOTE]
+====
+If you set a default node selector, the `oc debug node/<node>` command might not succeed. To work around this, first create a temporary project using `oc adm new-project` that overwrites the default node selector. Then, run the `oc debug node/<node>` command in that temporary project.
```

Comment:
Again `oc adm new-project <temporary_project> --node-selector=""` in the example.